post.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```

Comment:
you cheated there by including the info under the map with your map image! I wanted you to style that too :P